aiida/daemon/workflowmanager.py

```diff
@@ -40,10 +40,14 @@ def execute_steps():
     logger.info("Querying the worflow DB")
     
     running_steps = get_all_running_steps()
-    
+
     for s in running_steps:
         w = s.parent.get_aiida_class()
```

Comment:
A small comment to our discussion:
- I have profiled aiida.daemon.workflowmanager.execute_steps() and aiida.daemon.workflowmanager.advance_workflow() and it seems that we lose a lot of time in the generation of the the AiiDA object. This becomes an "issue" when launching a lot of workflows as N. Mounet does. I have a fix for this (also the code of the workflowmanager is a bit dirty since it uses in the same code backend objects and AiiDA orm objects without doing a clear distinction. However, I do not commit it now since it needs a bit more work to make sure that it works with SQLA.